app/models/bet.rb

```diff
@@ -2,4 +2,41 @@ class Bet < ApplicationRecord
   has_many :users, through: :user_bets
   has_many :user_bets
   has_many :options, through: :user_bets, source: :bet_option
+
+  def resolve
+    losers.each do |debtor|
+      debt = amount_bet(debtor)
+      win_contributions.each do |contribution|
+        debtor.debts.create(creditor: contribution[:creditor],
+                            amount:   debt * contribution[:percentage])
+      end
+    end
+  end
+
+  def win_option
+    options.winners
+  end
+
+  def winner_total
+    user_bets.winners.map(&:amount_bet).sum
+  end
+
+  def winners
```

Comment:
I dont think we need a helper method for this. Being able to type `winners` is not much different than typing `users.winners` in the code when you need it